webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/ElasticsearchBackupRepository.java

```diff
@@ -231,75 +280,33 @@ public List<GetBackupStateResponseDto> getBackups(final String repositoryName) {
     }
   }
 
-  @Override
-  public void executeSnapshotting(
-      final BackupService.SnapshotRequest snapshotRequest,
-      final Runnable onSuccess,
-      final Runnable onFailure) {
-    final var request =
-        new CreateSnapshotRequest()
-            .repository(snapshotRequest.repositoryName())
-            .snapshot(snapshotRequest.snapshotName())
-            .indices(snapshotRequest.indices())
-            // ignoreUnavailable = false - indices defined by their exact name MUST be present
-            // allowNoIndices = true - indices defined by wildcards, e.g. archived, MIGHT BE absent
-            .indicesOptions(IndicesOptions.fromOptions(false, true, true, true))
-            .includeGlobalState(backupProps.includeGlobalState())
-            .userMetadata(
-                objectMapper.convertValue(snapshotRequest.metadata(), new TypeReference<>() {}))
-            .featureStates(new String[] {"none"})
-            .waitForCompletion(true);
-    final var listener = new CreateSnapshotListener(snapshotRequest, onSuccess, onFailure);
-
-    esClient.snapshot().createAsync(request, RequestOptions.DEFAULT, listener);
-  }
-
-  private ActionListener<AcknowledgedResponse> getDeleteListener() {
-    return new ActionListener<>() {
-      @Override
-      public void onResponse(final AcknowledgedResponse response) {
-        LOGGER.debug(
-            "Delete snapshot was acknowledged by Elasticsearch node: {}",
-            response.isAcknowledged());
-      }
+  public void onDeleteResponse(final DeleteSnapshotResponse response) {}
 
-      @Override
-      public void onFailure(final Exception e) {
-        if (isSnapshotMissingException(e)) {
-          // no snapshot with given backupID exists, this is fine, log warning
-          LOGGER.warn("No snapshot found for snapshot deletion: {} ", e.getMessage());
-        } else {
-          LOGGER.error("Exception occurred while deleting the snapshot: {}", e.getMessage(), e);
-        }
-      }
-    };
-  }
+  public void onDeleteFailure(final Exception e) {}
 
   private boolean isSnapshotMissingException(final Exception e) {
-    return e instanceof ElasticsearchStatusException
-        && ((ElasticsearchStatusException) e)
-            .getDetailedMessage()
-            .contains(SNAPSHOT_MISSING_EXCEPTION_TYPE);
+    return e instanceof ElasticsearchException
```

Comment:
This is slightly different compared to the previous api, but there is no other field so it should be ok